pubsub/validators/validator.py

```diff
@@ -16,18 +29,21 @@ def validate_message(self, message):
         except AttributeError:
             raise AttributeError('Message must be json')
         schema_uri = message.get('schema', '')
-        matches = re.findall(r'(.+)://(.+/)?events/(.+)/(.+)/(.+)\.json', schema_uri)
-        if len(matches) >= 1:
-            schema = jsonschema.RefResolver('', '').resolve_remote(schema_uri)
-            jsonschema.validate(message, schema)
-        else:
-            raise ValidationError('Incorrect schema uri')
+        matches = re.findall(r'(.+)://(.+/)?(.+)\.json', schema_uri)
```

Comment:
There is probably a simpler way to assert URL using [`urlparse`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) or something.

Reply:
I looked into how Django validates URLs and it uses it's own vendored regex :Sob